wolox-books/src/app/sing-up/sing-up.component.html

```diff
@@ -0,0 +1,34 @@
+<main class="main-container">
+  <form [formGroup]="form" class="form-container">
+    <app-form (formSubmitted)="submitForm()" [isValid]="form.valid">
+      <app-input [label]="'Nombre'" [name]="'name'" [form]="form"></app-input>
+      <app-input
+        [label]="'Apellido'"
```

Comment:
you can remove the brackets and the single quotes. You should only use brackets when you are referring to something on the ts file